ekncontent/ekncontent/eknc-utils.c

```diff
@@ -203,6 +203,31 @@ eknc_get_ekn_version (const gchar *app_id,
   return g_strdup (stripped);
 }
 
+/**
+ * eknc_get_current_language:
+ *
+ * Gets the xapian friendly version of the current system language, or NULL if
+ * none set.
+ *
+ * Returns: (transfer none): the language
```

Comment:
nitpick: `transfer none` isn't necessary, it's inferred from `const gchar *` return type.